### ISSUE TITLE
[Question/Help] About the training code for MAMnet

### ISSUE THREAD

**Issue by coderbreake** [micahvista/MAMnet#11]
Hi there!

Thank you for sharing this excellent work and the code for your paper. I'm very interested in your research and have successfully run the MAMnet.py of the repository.

I'm currently trying to reproduce the full training pipeline as described in the paper. However, I couldn't locate the complete training script or the related configuration instructions in the README or the codebase.

Therefore, I would like to kindly ask: Would it be possible for you to share the complete training code for the MAMnet?

Best regards！

**Comment by micahvista** [comment]
Hi,
Thank you for your interest in our work and for reaching out.
I sincerely apologize for the inconvenience, but the original training code was stored on an school server that is no longer accessible. Consequently, I am unable to share the original training scripts.
However, you can still reproduce the training pipeline using the GIAB SV benchmark to label the data.
I hope this helps!
Best regards,
Hongyu

**Comment by micahvista** [comment]
Hi,

regarding the feature matrix, it can be produced using the baseinfo_AlignedSegment function. You can call it as follows:

baseinfo_AlignedSegment(genotype, bamfilepath, contig, r_start, r_end, meanvalue, window_size, maxcountread, workdir, INTERVAL)

Best regards,
Hongyu

**Comment by coderbreake** [comment]
Really appreciate you taking the time to explain this. It was a huge help!
